content/en/synthetics/api_tests/icmp_tests.md

```diff
@@ -79,7 +79,7 @@ When you set the alert conditions to: `An alert is triggered if any assertion fa
 
 #### Fast retry
 
-Your test can trigger retries in case of a failed test result. By default, the retries are performed 300 ms after the first failed test result. The retry interval can be configured with the [API][6].
+Your test can trigger retries in case of failed test result. Configuration is available to better suit your alerting sensibility.
```

Comment:
```suggestion
Your test can trigger retries `X` times after `Y` ms in case of a failed test result. Customize the retry interval to suit your alerting sensibility.
```